[Nara] Remove channel and video deletion by moderators mappings and CLI commands

zeeshanakram3 opened the pull request:
This PR addresses:

- #4740
- #4739
- Remove fixtures for above actions and fix integration tests

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1204404358291458/1205517269374078) by [Unito](https://www.unito.io)

kdembler committed:
[Argus] increase `cacheCleanup` in example config

mnaamani committed:
Merge pull request #4867 from kdembler/patch-2

[Argus] increase `cacheCleanup` in example config

zeeshanakram3 committed:
remove channel & video deletion cli commands

zeeshanakram3 committed:
remove channel & video deletion mappings

zeeshanakram3 committed:
fix integration tests

mnaamani reviewed (changes requested):
I don't think we should drop the mappings from the query-node because we need it to handle those events from previous runtimes, when we are processing blocks from genesis.

I left a comment about this in https://github.com/Joystream/joystream/pull/4874
Of course I also tried to fix the integration tests in that same PR and couldn't figure out why they were failing

https://github.com/Joystream/joystream/actions/runs/6218658040/job/16875580326?pr=4874#step:11:2141

Clearly you resolved it here :)

mnaamani commented:
Also this needs to be rebased onto nara branch

zeeshanakram3 committed:
Merge remote-tracking branch 'upstream/nara' into nara_remove_channel_and_video_deletion_mappings_and_cli

zeeshanakram3 committed:
removed events deleted events from runtime

zeeshanakram3 committed:
udate chain metedata

zeeshanakram3 committed:
revert meppings removal change

zeeshanakram3 committed:
update @joystream/types package

zeeshanakram3 committed:
add hydra-typegen tgz dependency temporarly

zeeshanakram3 commented:
I have:

- Rebased the PR on `nara` branch
- removed the deleted events (`VideoDeletedByModerator` & `ChannelDeletedByModerator`) events from runtime
- Updated the chain metadata (there is a new metadata file `query-node/chain-metadata/2002.json`)
- Reverted the mappings deletion change.
- Updated the `@joystream/types` package based on new chain-metadata.
- I have added a _temporary_ tar dependency of the [hydra-typegen](https://github.com/Joystream/hydra/pull/531) fix to test the PR easily

zeeshanakram3 committed:
bump hyrda depenedencies

mnaamani commented on query-node/codegen/package.json:
```diff
@@ -11,6 +11,6 @@
   "dependencies": {
     "@joystream/hydra-cli": "5.0.0-alpha.4",
-    "@joystream/hydra-typegen": "5.0.0-alpha.4"
+    "@joystream/hydra-typegen": "file:../../joystream-hydra-typegen-v5.0.0-alpha.4.tgz"
```
We want to use the newly published [5.0.0-alpha.6](https://www.npmjs.com/package/@joystream/hydra-typegen/v/5.0.0-alpha.6)

zeeshanakram3 replied:
Yeah, sorry, figured out that I didn't save the file before adding the commit

zeeshanakram3 committed:
update hydra-typegen & hydra-cli dependencies

mnaamani commented:
Merged this work in https://github.com/Joystream/joystream/pull/4874
